src/Store.test.js

```diff
@@ -0,0 +1,81 @@
+import Store from "./Store"
+import sinon from "sinon"
+import fetchMock from "fetch-mock"
+import moment from "moment"
+
+describe("Store", () => {
+  beforeEach(() => process.env.REACT_APP_API_PATH = "http://tb-api.example.com")
+  afterEach(() => fetchMock.reset())
+
+  describe("#saveNote()", () => {
+    it("Saves the note draft", function () {
+      let clock = sinon.useFakeTimers()
+      let now = moment().format("YYYY-MM-DD HH:mm:ss")
+      fetchMock.post(`path:/api/v1.0/notes`, 4)
+
+      let store = new Store()
+      store.loadNotes = sinon.fake()
+      store.patient_id = 88
+      store.noteDraft = "Hello!"
+
+      store.saveNote().then(() => {
+        expect(store.loadNotes.calledOnce).toBeTruthy()
+        expect(store.noteDraft).toBe(null)
+      })
+
+      let serverReceivedRequest = fetchMock.called((url, { method, headers, body }) =>
```

Comment:
It would be nice to shorten this somehow,
and I don't think it would be too difficult to write some more human-readable helpers for it.

However, that's probably outside the scope of this PR.
It would be a good refactoring task down the road.